model/task_lifecycle_test.go

```diff
@@ -7310,7 +7310,7 @@ func TestUpdateBlockedDependencies(t *testing.T) {
 	}
 	assert.NoError(execTask.Insert())
 
-	assert.NoError(UpdateBlockedDependencies(ctx, []task.Task{tasks[0]}))
+	assert.NoError(UpdateBlockedDependencies(ctx, []task.Task{tasks[0]}, false))
```

Comment:
Do any of our tests ever test the 'true' version of this function? I didn't see any direct uses of using 'true' for updateAllDependencies but maybe there's a test that it propagates down to a call that passes in 'true', but if we don't have any maybe we should add some

Reply:
Some of the host termination job unit tests will hit [this line](https://github.com/evergreen-ci/evergreen/pull/8323/files#diff-6dd488a00c797fb521cd37e03c8869d4fb82c17ee75a9c79eec5648ed8f7a05bR229)